dragonchain/broadcast_processor/broadcast_processor.py

```diff
@@ -87,6 +86,27 @@ def chain_id_set_from_matchmaking_claim(claim: dict, level: int) -> Set[str]:
     return set(claim["validations"][f"l{level}"].keys())
 
 
+def get_l5_wait_time(chain_id: str) -> int:
+    #  L5s should have an endpoint that tells that
+    network_attr = {
+        "btc": {"confirmations": btc.CONFIRMATIONS_CONSIDERED_FINAL, "block_time": btc.AVERAGE_BLOCK_TIME, "delay_buffer": 5.0},
+        "eth": {"confirmations": eth.CONFIRMATIONS_CONSIDERED_FINAL, "block_time": eth.AVERAGE_BLOCK_TIME, "delay_buffer": 1.5},
+        "bnb": {"confirmations": bnb.CONFIRMATIONS_CONSIDERED_FINAL, "block_time": bnb.AVERAGE_BLOCK_TIME, "delay_buffer": 1.5},
+    }  # block_time is in seconds; delay_buffer multiplier is to account for network delays
+    try:
+        mm_config = matchmaking.get_registration(chain_id)
+        interchain_network = mm_config["network"]  # returns: "btc" / "eth" / "bnb"
+        broadcast_interval = mm_config["broadcastInterval"]  # returns: decimal value in hours
+        broadcast_interval = int(((broadcast_interval * 60) * 60))  # converts to int value in seconds
```

Comment:
why all the parens? Makes it a bit harder to read, especially considering order of operations doesn't matter for multiplication

Reply:
Fair point, was a stylistic choice but it makes more sense just to `* 3600` so I'll change it to that.